### PR TITLE
Geometry tests

### PR DESCRIPTION
**Pull request by Athight**
- tests for line.py
- added greedy frechett_distance calculation (for testing as point creation accuracy wasn't great, might be useful one day though)
- not sure why changes that should already be in the master branch from #3 showing here 🤷‍♂

### CODE REVIEW

File: allfed_spatial/features/conversions.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import geopandas as gpd
```
Comment (trorsty) [inline allfed_spatial/features/conversions.py]: Looks like this branch is out of synch with master. I'd advise just pulling from master to get up to date, fixing any merge conflicts. See history here https://github.com/allfed/allfed-spatial/network

When starting a new branch, I think best practice should be:
- make sure up to date with master
- branch off master directly

File: allfed_spatial/geometry/line.py [hunk 2]
```diff
@@ -7,13 +7,86 @@
 from allfed_spatial.geometry.common import closest
 
 
+def frechet_distance(points1, points2):
+    """Test the distance between two lines
+    Imagine if a person was walking their dog,
+    and the person has to follow one line and the dog has to follow the other,
+    how long does the leash need to be?
```
Comment (trorsty) [inline allfed_spatial/geometry/line.py]: Cool! I'd suggest updating "how long does the leash need to be?" to "what is the length of the shortest leash sufficient for both to traverse their separate paths?" to be a bit more clear

File: tests/test_geometry_line.py [hunk 3]
```diff
@@ -0,0 +1,450 @@
+import unittest
+import allfed_spatial.geometry.line as geometry_line
+from shapely.geometry import Point, LineString, MultiLineString
+from allfed_spatial.features.feature import Feature
+
+class LineBaseTest(unittest.TestCase):
```
Comment (trorsty) [inline tests/test_geometry_line.py]: Suggest adding a docstring here with something like "Base class for testing functions with work with line geometries, containing some useful helper functions"

File: allfed_spatial/features/conversions.py (hunk 1 above)
Comment (Athight) [reply allfed_spatial/features/conversions.py]: aha, i had to merge master in again, I must have missed something, thanks for pointing that out!
I'm happy to follow those best practices, it sounds like opening a new branch for every pull request would make the most sense.
Do i just delete a branch after it's merged into master? (and then create a new branch off master for the next chunk)

File: allfed_spatial/geometry/line.py (hunk 2 above)
Comment (Athight) [reply allfed_spatial/geometry/line.py]: updated

File: tests/test_geometry_line.py (hunk 3 above)
Comment (Athight) [reply tests/test_geometry_line.py]: added a short docstring

File: allfed_spatial/features/conversions.py (hunk 1 above)
Comment (trorsty) [reply allfed_spatial/features/conversions.py]: No problem! And yeah making a branch for each PR would be ideal, and delete after merge.

**Pull request merged**